Packs/GenetecSecurityCenter/Integrations/GenetecSecurityCenterEventCollector/GenetecSecurityCenterEventCollector.py

```diff
@@ -0,0 +1,176 @@
+import demistomock as demisto
+from urllib3 import disable_warnings
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+from requests.auth import HTTPBasicAuth
+import json
+from xmltodict import parse
+
+disable_warnings()
+
+""" CONSTANTS """
+
```

Comment:
add at least 1 json file under test_data that has a raw response that can be used as an example